src/gas-price/gas-price.ts

```diff
@@ -161,15 +161,16 @@ export const getRecommendedGasPrice = async (
   const lastUpdateTimestamp = sponsorLastUpdateTimestamp[sponsorWalletAddress];
 
   // Check if the next update is a retry of a pending transaction and if it has been pending longer than scalingWindow
-  if (lastUpdateTimestamp && lastUpdateTimestamp < Math.floor(Date.now() / 1000) - scalingWindow) {
+  if (lastUpdateTimestamp) {
+    const pendingPeriod = Math.floor(Date.now() / 1000) - lastUpdateTimestamp;
     const multiplier = calculateScalingMultiplier(
       recommendedGasPriceMultiplier,
       maxScalingMultiplier,
```

Comment:
It's nit and not related to objective of this PR but I'd suggest renaming `maxScalingMultiplier` to `maxGasPriceMultiplier` or `maxRecommendedGasPriceMultiplier`. The current name, `maxScalingMultiplier`, doesn't convey the intended meaning effectively; it could be misinterpreted as a multiplier for `recommendedGasPriceMultiplier`.

Reply:
Hehe, I actually was thinking similarly [here](https://github.com/api3dao/airseeker-v2/pull/224/commits/66cf9152da5989f87a86a1a3fa29e1bcf1f95d86#diff-a25ae601baae9bca444c00d66e4934596768a26eafd5208c8fa077dcf31b4ae9R40)
 but at the end didn't do it, because it would be a breaking change and I wasn't sure if I am not too subjective.
 
At the end I also accepted `maxScalingMultiplier` as a good name, because it's only applied during gas price scaling. I still think `baseGasPriceMultiplier` would be a better name compared to `recommendedGasPriceMultiplier`

Reply:
I feel like it's not worth doing (nor opening an issue for). If you feel otherwise, feel free to create the issue.